[xlnt] XLNT_STATIC macro redefinition

**Describe the bug**
On windows planform there is clang-tidy \ build warning

**Environment**
- Windows 10
- Compiler: MSVC 19

**To Reproduce**
Just use lib in the project. set warnings level to max

**Expected behavior**
Clean build with no warnings

**Failure logs** 
[build] vcpkg_installed\x64-windows-static\include\xlnt/xlnt.hpp(25): warning C4005: 'XLNT_STATIC': macro redefinition

**Additional context**
Bug introduced by patching lib by vcpkg. Valid only to windows platform. Patch introduced to lib version 1.4. Current version if 1.5 and this patch is not needed anymore.

--- a/port_versions/baseline.json
+++ b/port_versions/baseline.json
@@ -6162,7 +6162,7 @@
     },
     "xlnt": {
       "baseline": "1.5.0",
-      "port-version": 0
+      "port-version": 1
     },
     "xmlsec": {
       "baseline": "1.2.31",

--- a/port_versions/x-/xlnt.json
+++ b/port_versions/x-/xlnt.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "1e9127d56c2d772d64454b5aef80f93689f70242",
+      "version-string": "1.5.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "1f17fd4b98bd2f8a5f0d02aa8db2cd7b1ed1552f",
       "version-string": "1.5.0",
       "port-version": 0

--- a/ports/xlnt/CONTROL
+++ b/ports/xlnt/CONTROL
@@ -1,4 +1,5 @@
 Source: xlnt
 Version: 1.5.0
+Port-Version: 1
 Homepage: https://github.com/tfussell/xlnt
 Description: Cross-platform user-friendly xlsx library for C++14

--- a/ports/xlnt/static-linking-for-windows.patch
+++ b/ports/xlnt/static-linking-for-windows.patch
@@ -1,13 +1,16 @@
 diff --git a/include/xlnt/xlnt.hpp b/include/xlnt/xlnt.hpp
-index 01fe4981..9cefa6e0 100644
+index fdd33c38..0d21128f 100644
 --- a/include/xlnt/xlnt.hpp
 +++ b/include/xlnt/xlnt.hpp
-@@ -22,7 +22,7 @@
- // @author: see AUTHORS file
+@@ -23,6 +23,11 @@
  
  #pragma once
--
+ 
++// Patched by vcpkg to handle static non-cmake builds
++#ifndef XLNT_STATIC
 +#define XLNT_STATIC
++#endif
++
  #include <xlnt/xlnt_config.hpp>
  
  // cell

